
<launch>
    <node type="usb_cam_node" name="usb_cam" pkg="usb_cam"
        output="screen" args="_video_device:=/dev/video0 _pixel_format:=yuyv _camera_name:=tracker_camera"
    />
</launch>
